src/mintpy/objects/stack.py

```diff
@@ -1164,20 +1164,20 @@ def get_sequential_closure_phase(self, box, conn, post_proc=None):
                 raise Exception(f"No triplets found at connection level: {conn}!")
 
         ## read data
-        phase = self.read(box=box, print_msg=False)
-        ref_phase = self.get_reference_phase(dropIfgram=False)
-        for i in range(phase.shape[0]):
-            mask = phase[i] != 0.
-            phase[i][mask] -= ref_phase[i]
```

Comment:
If I understood it right, spatial referencing and masking are needed if using `unwrapPhase`.

Reply:
Spatial referencing is not needed. Since no moisture change should have a naturally zero closure phase, choosing a reference point in the wrong place can lead to misinterpretations. I tested in Central Valley and proved spatial referencing is not needed.
Not sure about the masks. Why do we need masks here in the first place?

Reply:
Thank you for your explanation offline. I agree with you that spatial referencing is not needed here. Since masking is serving the referencing process, it's also not needed anymore.